docs/integration_resources/software/nuke.md

```diff
@@ -6,17 +6,84 @@ Shift provides a `nshift` utility script to properly bind the Shift UI to Nuke.
 
 ```python
 import sys
-sys.path.append("<path_to_your_shift_folder>")
-sys.path.append("<path_to_your_shift_folder>/thirdparty/python/Lib/site-packages")
+sys.path.append("<path_to_the_shift_installation_folder>")
+sys.path.append("<path_to_the_shift_installation_folder>/shift/thirdparty/python/Lib/site-packages")
 
 from shift.tools import nshift
 nshift.show()
 ```

```

Comment:
Update to use code block formatting 
https://github.com/Inbibo/shift_docs/issues/60#issuecomment-2326086193